### PR TITLE
add enkidu for unist

### PR DESCRIPTION
**Pull request by yithian**

### CODE REVIEW

**Comment by yithian** [comment]
for #48

File: data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+      {
+        "name": "Veil Off",
+        "inputs": [
+          "A+B+C"
+        ]
+      }
+    ],
+    [
+      {
+        "name": "First Prohibition",
```
Comment (komidore64) [inline data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json]: Let's also give First Prohibition and Second Prohibition `shortName` fields. The way follow ups are rendered you get some ugly word wrap on long names. Maybe "1st Pro" and "2nd Pro" or something?

File: data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+              {
+                "name": "Third Prohibition: Pierce",
+                "inputs": [
+                  "6A"
+                ]
+              }
+            ]
+          },
+          {
+            "name": "Second Prohibition (Secret): Sever",
```
Comment (komidore64) [inline data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json]: `shortName` here too?

File: data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json (hunk 1 above)
Comment (yithian) [reply data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json]: That makes sense.

I just noticed that in the spec, we have `follow_ups` and `shortName`. Was it intentional to use both camel case and underscores? Should we standardize on one or the other?

Comment (komidore64) [reply data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json]: I would like to standardize, I just haven't fixed `follow_ups` to `followUps` yet.

Comment (komidore64) [reply data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json]: I created #58 for tracking.

**Comment by komidore64** [comment]
please2rebase

**Comment by yithian** [comment]
rebased and fixed conflict

**Pull request merged**